packages/analytics-browser/test/config/joined-config.test.ts

```diff
@@ -105,17 +117,98 @@ describe('joined-config', () => {
         };
         mockRemoteConfigFetch = {
           getRemoteConfig: jest.fn().mockResolvedValue(remoteConfig),
-          fetchTime: fetchTime,
+          metrics: metrics,
         };
         // Mock the createRemoteConfigFetch to return the mockRemoteConfigFetch
         (createRemoteConfigFetch as jest.MockedFunction<typeof createRemoteConfigFetch>).mockResolvedValue(
           mockRemoteConfigFetch,
         );
 
+        const expectedAutocapture = {
+          fileDownloads: false,
+          formInteractions: false,
+          pageViews: false,
+          attribution: false,
+          sessions: false,
+          elementInteractions: true,
+        };
+
         await generator.initialize();
         expect(generator.config.defaultTracking).toBe(true);
+        expect(generator.config.autocapture).toBe(true);
         const joinedConfig = await generator.generateJoinedConfig();
-        expect(joinedConfig.defaultTracking).toBe(remoteConfig.autocapture);
+        expect(joinedConfig.defaultTracking).toStrictEqual(expectedAutocapture);
+        expect(joinedConfig.autocapture).toStrictEqual(expectedAutocapture);
+      });
+
+      test.each([
+        {
+          sessions: false,
+          fileDownloads: false,
+          formInteractions: false,
+          attribution: false,
+          pageViews: false,
+        },
+        false,
+      ])('should only enable elementInteractions if remote config only sets it to true', async (localAutocapture) => {
```

Comment:
Can we change the description?  `elementInteractions: false,` in the remote config. You are actually test, the 
elementInteractions is false when  remote config only sets it to false.

Reply:
I don't see much difference here. What will be the description you are suggestion here?

Reply:
`should only disable elementInteractions if remote config only sets it to false'`

Reply:
Nice catch! I change the remote config value to match the description at https://github.com/amplitude/Amplitude-TypeScript/pull/848/commits/22de814075cde2d7ed9cadee444e04aa9cab5f4d